recipe/conda_forge_ci_setup/feedstock_outputs.py

```diff
@@ -97,19 +98,20 @@ def is_valid_feedstock_output(project, outputs):
             _, o, _, _ = split_pkg(dist)
         except RuntimeError:
             continue
-        
+
         for i in range(3):  # three attempts
             try:
                 registered_feedstocks = package_to_feedstock(o)
             except requests.HTTPError as exc:
                 if exc.response.status_code == 404:
                     # no output exists and we can add it
+                    # if we turn off autoregistration, then the True here should be False
                     valid[dist] = True
```

Comment:
@isuruf This PR does not switch the default yet. If you'd rather have it do that and wait, that is fine.